changelog.json

```diff
@@ -1,5 +1,16 @@
 {
   "releases": {
+    "1.4.0-beta1": [
+      "[New] Release notes for update are now displayed within Desktop - #2774",
```

Comment:
`Release notes are now displayed within Desktop`?

Reply:
I'm trying to scope this correctly to avoid the idea that _all_ release notes will be displayed in the dialog. What about:

> `Release notes for next update are displayed within Desktop`?

Reply:
`Release notes for Desktop app updates are displayed within Desktop`?